### PR TITLE
Ingest adapter info from into snapshot

### PR DESCRIPTION
**Pull request by guptasu**
* Validate the adapter configuration for message name "Param".
* Copy the ingested configuration into snapshot object.

### CODE REVIEW

File: mixer/pkg/runtime/config/snapshot.go [hunk 1]
```diff
@@ -39,6 +39,9 @@ type (
 		Instances map[string]*Instance
 		Rules     []*Rule
 
+		Templates2 map[string]*TemplateMetadata
```
Comment (mandarjog) [inline mixer/pkg/runtime/config/snapshot.go]: comment?

Comment (guptasu) [reply mixer/pkg/runtime/config/snapshot.go]: D

**Comment by guptasu** [comment]
@geeknoid re-lgtm please

**Pull request merged**